assets/src/design-system/theme/colors.js

```diff
@@ -65,46 +98,161 @@ const standard = {
   white: '#FFF',
 };
 
+const opacity = {
+  footprint: 'transparent',
+  white64: rgba(standard.white, 0.64),
+  white24: rgba(standard.white, 0.24),
+  white16: rgba(standard.white, 0.16),
+  black64: rgba(standard.black, 0.64),
+  black24: rgba(standard.black, 0.24),
+  black10: rgba(standard.black, 0.1),
+  blue24: rgba(brand.blue[30], 0.24),
+  violet24: rgba(brand.violet[30], 0.24),
+  overlay: rgba('#1E1F1F', 0.5),
```

Comment:
@zachhale good question about the modal overlay. The comment I am remembering is herehttps://github.com/google/web-stories-wp/issues/4956\#issuecomment-719950944 and i can't remember why i swapped this for the opacity.black64 so I added back in this overlay value so that when this goes through UAT sam and co can verify.  

thank you for your detailed look at this! SO.MANY.COLORS!

Reply:
Great!